application/backend/app/api/endpoints/training_configurations.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, HTTPException, status
+
+from app.api.dependencies import get_training_configuration_service
+from app.configuration_tools.training_configuration_converter import convert_training_configuration_to_rest
+from app.services import ResourceNotFoundError
+from app.services.training_configuration_service import TrainingConfigurationService
+
+logger = logging.getLogger(__name__)
+router = APIRouter(prefix="/api/projects/{project_id}/training_configuration", tags=["Training Configuration"])
+
+
+@router.get("")
+def get_training_configuration(
```

Comment:
This response is malformed, because all training parameters are flattened under "training".

<img width="369" height="590" alt="image" src="https://github.com/user-attachments/assets/e4e048b6-4d18-4134-bb21-8911fcd5e18d" />

See how it differs from the response in Geti Classic:

<img width="580" height="330" alt="image" src="https://github.com/user-attachments/assets/e503a340-cc2a-4026-9ca0-4c918fc3a142" />

Here "enable" and "patience" are correctly nested under "early_stopping".